pages/install/conda-install.rst

```diff
@@ -45,32 +47,87 @@ obtained are specified by adding the ``--channel`` option in the ``install`` or
 ``cantera`` channel, so we add ``--channel cantera`` and to tell Conda to look at the
 ``cantera`` channel in addition to the default channels.
 
-To use the scripts and modules installed in the ``spam`` environment,
-you must activate it it by running::
+To use the scripts and modules installed in the ``ct-env`` environment, including Jupyter,
+you must activate it it by running:
+
+.. code:: shell
+
+   conda activate ct-env
+
+**Option 2: Create a new environment using an environment file**
+
+This option is similar to **Option 1** but includes a few other packages that
+you may find helpful as you're working with Cantera. Copy and paste the contents
+of the file shown below into a file called ``environment.yaml``. Then, save the
+the file somewhere and remember that location.
+
+.. code:: yaml
+
+   name: ct-env
+   channels:
+   - cantera
+   - defaults
+   dependencies:
+   - python
+   - cantera
+   - ipython  # optional (needed for nicer interactive command line)
+   - jupyter  # optional (needed for Jupyter Notebook)
+   - matplotlib  # optional (needed for plots)
+   - python-graphviz  # optional (needed for reaction path diagrams)
+   - h5py  # optional (needed for HDF/H5 output)
+   - pandas  # optional (needed for pandas interface)
+
+From the command line (or the
+Anaconda Prompt on Windows), change directory into the folder where you saved
+``environment.yaml``:
+
+.. code:: shell
+
+   cd folder/where/you/saved
 
-    conda activate spam
+and then run:
 
-**Option 2: Install Cantera in an existing environment**
+.. code:: shell
 
-First, activate your environment (assumed here to be named ``baked_beans``; if you've
+   conda env create -f environment.yaml
+
+This will create an environment called ``ct-env``. Once you've done that, you
+need to activate the environment before using any scripts or modules that you
+just installed:
+
+.. code:: shell
+
+   conda activate ct-env
+
+**Option 3: Install Cantera in an existing environment**
```

Comment:
Do we want to encourage this? There is no real benefit for people familiar with `conda` (`environment.yml. already contains all dependencies that need to be known), while others may get tangled up quite a bit.

Reply:
That's a good point, I think we can remove this option.